deploy/kubernetes/service/mongodb/statefulset-dev.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: mongo
+spec:
+  serviceName: "mongo"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        service: mongo
+        role: db
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+        - name: mongo
+          resources:
+            requests:
+              cpu: 100m
+          image: mongo
+          command:
+            - mongod
+            - "--bind_ip"
+            - 0.0.0.0
+          ports:
+            - containerPort: 27017
+          volumeMounts:
+            - name: mongodb
+              mountPath: /data/db
+  volumeClaimTemplates:
+  - metadata:
+      name: mongodb
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
+      storageClassName: "manual"
+      resources:
+        requests:
+          storage: 10Gi
```

Comment:
Duplicate mongo/statefulset 
dev -> manual for local testing
default -> slow-many for production